app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -469,9 +472,15 @@ class StatePlayerRecyclerViewAssembler private constructor(
     val confettiConfig = checkNotNull(congratulationsTextConfettiConfig) {
       "Expected non-null reference to confetti animation configuration"
     }
-
     createBannerConfetti(confettiView, confettiConfig)
     animateCongratulationsTextView(textView)
+
+    if (feedback.html.isBlank()) {
```

Comment:
Can you please also update the KDoc to explain what feedback is & how it's used (for context to the readers on how to specify its value)?

Reply:
Done.